contrib/hashicorp/vault/example_test.go

```diff
@@ -0,0 +1,95 @@
+package vault_test
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/contrib/hashicorp/vault"
+
+	"github.com/hashicorp/vault/api"
+)
+
+// This is the most basic way to enable tracing with Vault
+func ExampleNewHTTPClient() {
+	c, err := api.NewClient(&api.Config{
+		HttpClient: vault.NewHTTPClient(),
+		Address:    "http://vault.mydomain.com:8200",
+	})
+	if err != nil {
+		log.Fatalf("Failed to create vault client: %s\n", err)
+	}
+
+	// This call wil be traced
+	c.Logical().Read("/secret/key")
+}
+
+// Options can be passed in to configure the tracer
+func ExampleNewHTTPClient_withOptions() {
+	c, err := api.NewClient(&api.Config{
+		HttpClient: vault.NewHTTPClient(
+			vault.WithServiceName("my.vault"),
+			vault.WithAnalytics(true),
+			vault.WithAnalyticsRate(1.0)),
+		Address: "http://vault.mydomain.com:8200",
+	})
+	if err != nil {
+		log.Fatalf("Failed to create vault client: %s\n", err)
+	}
+
+	// This call wil be traced
+	c.Logical().Read("/secret/key")
+}
+
+// If you already have an *http.Client that you're using (c in this example), you can continue to use it by
```

Comment:
I think the parenthesis is not needed, because you have inline comments in the code making it clear.
```suggestion
// If you already have an *http.Client that you're using, you can continue to use it by
```